localstack/services/kms/models.py

```diff
@@ -156,15 +161,17 @@ class KmsCryptoKey:
     public_key: Optional[bytes]
     private_key: Optional[bytes]
     key_material: bytes
+    key_spec: str
 
     def __init__(self, key_spec: str):
         self.private_key = None
         self.public_key = None
+        self.key_material = os.urandom(SYMMETRIC_DEFAULT_MATERIAL_LENGTH)
+        self.key_spec = key_spec
```

Comment:
We need to add the following comment: 

> Technically, key_material, being a symmetric encryption key, is only relevant for
> key_spec == SYMMETRIC_DEFAULT.
> But LocalStack uses symmetric encryption with this key_material even for other specs. Asymmetric keys are
> generated, but are not actually used for encryption. Signing is different.

above this code.

Reply:
Moved :+1: